### ISSUE TITLE
UXCam isn't capturing most of my app, corrupted/incorrect recordings

### ISSUE THREAD

**Issue by evelant** [uxcam/react-native-ux-cam#7]
I just integrated uxcam in my react-native 0.63.2 app but it's failing to record correctly most of the time. The issue is difficult to describe. 

It seems "stuck" on some frames. The recording doesn't update as the user interacts with the app, it only shows a static image of a dialog that either was never shown to the user or that they had already dismissed. Sometimes at seemingly random points the recording "updates" and shows the content of the screen correctly for one frame but is then frozen again and continues to miss all of the changes as the user interacts with the app. 

The user taps appear to be recorded properly, it's just the recorded view doesn't even remotely match what's actually shown on the user's device. I've tested on the Android studio emulator and a physical Android device both in release mode and both with the same results.

Here's the dependencies we're using
```
        "@notifee/react-native": "^0.12.2",
        "@react-native-community/async-storage": "^1.12.0",
        "@react-native-community/google-signin": "^4.0.3",
        "@react-native-community/netinfo": "^5.9.5",
        "@react-native-firebase/analytics": "^7.6.1",
        "@react-native-firebase/app": "^8.4.1",
        "@react-native-firebase/auth": "^9.1.2",
        "@react-native-firebase/crashlytics": "^8.4.2",
        "@react-native-firebase/firestore": "^7.7.0",
        "@react-native-firebase/functions": "^7.4.2",
        "@react-native-firebase/messaging": "^7.8.4",
        "@react-native-firebase/perf": "^7.4.2",
        "clevertap-react-native": "^0.4.5",
        "date-fns": "^2.15.0",
        "fast-safe-stringify": "^2.0.7",
        "instabug-reactnative": "^9.1.7",
        "lodash": "^4.17.20",
        "metro-minify-terser": "^0.63.0",
        "mobx": "^5.15.4",
        "mobx-react": "^6.2.5",
        "mobx-state-tree": "^3.17.2",
        "mobx-utils": "^5.6.1",
        "moment": "^2.27.0",
        "observable-hooks": "^3.1.0",
        "react": "16.13.1",
        "react-native": "0.63.2",
        "react-native-animatable": "^1.3.3",
        "react-native-branch": "^4.4.0",
        "react-native-code-push": "^6.3.0",
        "react-native-device-log": "github:AndrewMorsillo/react-native-device-log#master",
        "react-native-exception-handler": "^2.10.8",
        "react-native-fbsdk": "^1.1.2",
        "react-native-popover-view": "^3.0.3",
        "react-native-restart": "^0.0.17",
        "react-native-safe-area-context": "^3.1.6",
        "react-native-sound": "^0.11.0",
        "react-native-svg": "^12.1.0",
        "react-native-ux-cam": "^5.2.0",
        "react-native-vector-icons": "^7.0.0",
        "react-native-version-number": "^0.3.6",
        "react-native-xyz-scrollview": "github:AndrewMorsillo/react-native-directed-scrollview#master",
        "rxjs": "^6.6.2",
        "seedrandom": "^3.0.5",
        "tinycolor2": "^1.4.1"
```

This unfortunately makes uxcam 100% unusable.  Our app is very animation heavy and all animations are using `useNativeDriver:true`, maybe uxcam can't record animated views properly on react-native?

**Comment by susanuxcam** [comment]
Hey @AndrewMorsillo, thanks for reporting this! We've already got this request on the chat inside our dashboard and we created a ticket for our developers. We will keep you posted!

**Comment by tomwanzek** [comment]
Has there ever been a follow-up/resolution to this issue?

**Comment by tomwanzek** [comment]
Specifically, we are interested on whether there are any problematic interactions having UXCam and Instabug in the same code base. As the package.json seems to indicate both were used in the App for this post. I.e. wondering in light of this recent post by Instabug: https://help.instabug.com/en/articles/2102240-can-i-use-the-instabug-crash-reporting-tool-in-parallel-with-other-crash-reporters 

Note: We are also using Firebase (analytics, messaging), but not chrashlytics. So it would be only UXCam and Instabug both tapping crash reporting to different degrees.

**Comment by evelant** [comment]
@tomwanzek  We gave up on UXCam since it never worked right on react-native. I would also warn against using Instabug. In our experience it was a large source of instability on react native Android (but worked flawlessly on iOS). We had session crash rates in excess of 7% all of it coming from Instabug. The issues logged with Instabug about the instability went unresolved for a long time and AFAIK are still unresolved. We abandoned it for another solution.
